log-collector-script/linux/eks-log-collector.sh

```diff
@@ -509,6 +515,18 @@ get_sysctls_info() {
   ok
 }
 
+get_network_policy_ebpf_info() {
+  try "collect network policy ebpf loaded data"
+  echo "*** EBPF loaded data ***" >> "${COLLECT_DIR}"/networking/ebpf-data.txt
+  LOADED_EBPF=$(/opt/cni/bin/aws-eks-na-cli ebpf loaded-ebpfdata | tee -a "${COLLECT_DIR}"/networking/ebpf-data.txt)
```

Comment:
Should we test for the presence of this binary `aws-eks-na-cli` before using it?

Reply:
I see the related discussion here https://github.com/awslabs/amazon-eks-ami/pull/1595/files

Reply:
Sine this is already merged, not sure why this is here in this diff.

Reply:
Looks like this came from either Carter committing directly to this branch or fallout from the name of the primary branch being changed to `main`. Either way, I will delete and recreate this PR to fix it